references/js-callout/apiproxy/resources/jsc/responseMediation.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const responseBody = JSON.parse(context.getVariable('response.content'));
+
+if (responseBody.headers['X-Amzn-Trace-Id']) {
```

Comment:
just a thought... what if httpbin removes this header?

i suggest we leave it as it is and look to use an apigee mock in future where we control the headers